pom.xml

```diff
@@ -978,8 +997,7 @@
                 <spark.major.version>spark2.2</spark.major.version>
                 <zookeeper.version>3.4.5-cdh5.14.0</zookeeper.version>
                 <spark.version>2.2.0.cloudera1</spark.version>
```

Comment:
If all cdh platforms are upgraded to 2.2.0.cloudera2, why leave this one at 2.2.0.cloudera1?
If 2.1.0.cloudera1, 2.2.0.cloudera1 are gone, hbase_sql/spark2.1.0.cloudera1, hbase_sql/spark2.2.0.cloudera1 should be removed too.